libvast/src/type.cpp

```diff
@@ -459,6 +459,11 @@ type::type(std::string_view name, const type& nested,
         attributes_offsets.emplace_back(fbs::type::detail::CreateAttribute(
           builder, key_offset, value_offset));
       };
+      std::sort(attributes.begin(), attributes.end(),
```

Comment:
I have heard that this file is very optimized. I am not sure what implications it has. Can someone more experienced confirm that this is ok? :D

Reply:
Yes, this is actually a required bug fix. The size of this vector is 0 or 1 most of the time, so it's not critical.